src/Stripe.net/Entities/_base/StripeEntity.cs

```diff
@@ -25,9 +31,15 @@ public abstract class StripeEntity : IStripeEntity
         /// </remarks>
         /// <returns>The raw <see cref="JObject">JObject</see>.</returns>
         [JsonIgnore]
+#if NET6_0_OR_GREATER
+        [STJS.JsonIgnore]
+#endif
         public JObject RawJObject { get; protected set; }
```

Comment:
consider throwing a nice error.

Reply:
I thought about this some more.  The problem is that once we serialize the object to JSON, we lose access any unsupported fields that would have been only on the API response but not deserialized into a C# property.  If a user were to deserialize from the serialized JSON, there would be no value to accessing these fields via RawJObject.  I like the idea of reporting an error but STJ would require us to put a converter on every concrete type in order for it to be effective.  I think the best course for now is a comment here, and a section in the readme describing what the support looks like and what its limitations are, and then we wait for customer feedback.